sdk/python/feast/registry_store.py

```diff
@@ -0,0 +1,39 @@
+from abc import ABC, abstractmethod
+
+from feast.protos.feast.core.Registry_pb2 import Registry as RegistryProto
+
+
+class RegistryStore(ABC):
+    """
+    RegistryStore: abstract base class implemented by specific backends (local file system, GCS)
```

Comment:
This comment can be made tighter. We already know the class is `RegistryStore`, and an abstract base class, so that doesn't need to be repeated.

What about

```
A registry store is a storage backend for the Feast registry.
```